okhttp/src/main/java/io/grpc/okhttp/OkHttpChannelBuilder.java

```diff
@@ -322,6 +368,8 @@ public final OkHttpChannelBuilder usePlaintext() {
    */
```

Comment:
For line 366: it will be good to fix the link to `{@link #sslSocketFactory(SSLSocketFactory)}`

Reply:
That's really-really preexisting. But done.